OuterWildsRandomSpeedrun/OuterWildsRandomSpeedrun.cs

```diff
@@ -170,14 +136,15 @@ private void CreateTimer()
 
         private void HandleBasicWarp(PlayerSpawner spawner, SpawnPoint[] spawnPoints)
         {
-            if (_spawnPointName == null || _goalPointName == null)
+            if (SpeedrunState.INSTANCE.SpawnPointId == null || SpeedrunState.INSTANCE.GoalPointId == null)
```

Comment:
Oops this should be deleted, but it's effectively dead code at the moment. Will fix in a followup.